nodejs/eks/nodegroup.ts

```diff
@@ -841,14 +894,8 @@ function createNodeGroupInternal(
                 .all([nodeSecurityGroupId, extraNodeSecurityGroupIds])
                 .apply(([sg, extraSG]) => [sg, ...extraSG]),
             spotPrice: args.spotPrice,
-            rootBlockDevice: {
```

Comment:
I wish we would've just used `ebsBlockDevices` here. That would've simplified this a lot...
To stay backwards compatible we now have to do this if/else dance above :(